pages/agent/v3/hooks.md.erb

```diff
@@ -82,3 +87,19 @@ HOOK_SCRIPT
 
 chmod +x .buildkite/hooks/pre-command
 ```
+
+## Hooks on Windows
+
+Agents runnings Windows require hook files to have either:
+
+* A `.bat` extension, and be written in [Windows Batch](https://en.wikipedia.org/wiki/Batch_file), or 
+* Access to <a href="windows#git-for-windows">Git for Windows, which includes bash</a>
```

Comment:
I think the link here should be `/docs/agent/v3/windows`?

Reply:
And ` <a href="windows#git-for-windows">Git for Windows, which includes bash</a> ` should probs just be ` <a href="windows#git-for-windows">Git for Windows</a> , which includes bash `